wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -288,12 +287,16 @@ class KotlinGenerator private constructor(
     type.fieldsWithJavaInteropOneOfs().forEach { field ->
       val fieldName = nameAllocator[field]
 
+      val propertyBuilder = PropertySpec.builder(fieldName, field.declarationClass)
+          .mutable(true)
+          .initializer(field.getDefaultValue())
+
+      if (javaInterOp) {
+        propertyBuilder.addAnnotation(JvmField::class)
+      }
+
       builder
-          .addProperty(PropertySpec.builder(fieldName, field.declarationClass)
```

Comment:
Pretty sure this code only runs in interop, since I don't see any changes to non-interop tests. Would it make sense to just change the code in place?

Reply:
According to this:
```
  private fun MessageType.fieldsWithJavaInteropOneOfs(): List<Field> {
    return if (javaInterOp)
      fieldsAndOneOfFields()
    else
      fields()
  }
```

this code could still run in non-interop mode.